src/renderer/components/ft-list-video/ft-list-video.vue

```diff
@@ -45,7 +45,7 @@
         @click="handleExternalPlayer"
       />
       <ft-icon-button
-        v-if="!isLive"
+        v-if="!isUpcoming"
```

Comment:
Just curious, do we want to stop upcoming videos from being saved? I think for a "watch later" playlist it could make sense to be able to add upcoming videos (if it doesn't cause any other issues, of course)

Reply:
Feel free to open PR (I am just fixing consistency issue here)